### DISCUSSION TITLE
custom sharpness

### DISCUSSION THREAD

**Discussion by lyricsdude** [gopro/labs#921] [Feature Requests]
is it possible to customize sharpness with a parameter in the future with a gopro labs command?
because in-body sharpening is different than sharpening in post (dealing with aliasing and lighting up noise), and the medium sharpness setting is too harsh still...

we can control the noise reduction in gopro labs, but tweaking the noise reduction influences the sharpness

thank you :)

**Comment by dnewman-gpsw** [comment]
It is a good idea.  Added to the feature list to investigate.

**Comment by xiemeilong** [comment]
Two years have passed, and we still can't turn off sharpening.

**Comment by dnewman-gpsw** [comment]
Low sharpening is too match?  It is not something I have found.

**Comment by xiemeilong** [comment]
I intend to use GoPro video as input data for direct Visual-Inertial Odometry (VIO). Therefore, it is necessary to disable sharpening to maintain photometric consistency. However, I haven't tested yet whether the Low sharpening setting is still unacceptable.

**Reply by dnewman-gpsw** [reply]
Others have done this and I haven't heard of any issue with low sharpness
